core/tests/org.fusesource.ide.camel.model.service.core.tests.integration/src/main/resources/withMarshalCsvHeaders.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.osgi.org/xmlns/blueprint/v1.0.0 https://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd                            http://camel.apache.org/schema/blueprint http://camel.apache.org/schema/blueprint/camel-blueprint.xsd">
+    <camelContext id="_context1" xmlns="http://camel.apache.org/schema/blueprint">
+        <route id="extract-route">
+                     <from id="cron" uri="{{extract.schedule}}"/>
+            <transacted id="_transacted" ref="PROPAGATION_REQUIRED"/>
+            <log id="log_start" message="Firing db call"/>
+            <to id="to_select" uri="sql:classpath:cx_extract_combined.sql"/>
+            <to id="to_update" uri="sql:{{sql.cx.update}}"/>
+            <marshal id="marshal">
+                <csv delimiter="|">
+                    <header>person_id</header>
+                    <header>first_name</header>
+                    <header>last_name</header>
+                    <header>email_address</header>
+                    <header>collar_number</header>
+                    <header>user_id</header>
+                    <header>ex_employee</header>
+                    <header>post_rank</header>
+                    <header>job_title</header>
+                </csv>
+            </marshal>
+            <to id="to_local_file" uri="{{local.generated.file.path}}?fileName=cx_export.csv.${date:now:yyyy-MM-dd-HHmmss}&amp;tempPrefix=."/>
+              </route>
+    </camelContext>
+</blueprint>
```

Comment:
in the file there is no comma, it is only a quick fix for the list of values inside the header fields